### PR TITLE
DOC: misleading example in docstring

### PR DESCRIPTION
**Pull request by jasmainak**

### CODE REVIEW

File: mne/evoked.py [hunk 1]
```diff
@@ -377,7 +377,7 @@ def plot(self, picks=None, exclude='bads', unit=True, show=True, ylim=None,
         show : bool
             Call pyplot.show() at the end or not.
         ylim : dict
-            ylim for plots. e.g. ylim = dict(eeg=[-200e-6, 200e-6])
+            ylim for plots. e.g. ylim = dict(eeg=[-200, 200])
```
Comment (jona-sassenhagen) [inline mne/evoked.py]: Shouldn't that be `-200e-6. 200e-6`?

Also, see line 438 for a similar, but different error :)

Comment (jasmainak) [inline mne/evoked.py]: doesn't seem to work for me. Not sure if it needs a fix in the docstring or the logic

Comment (jona-sassenhagen) [inline mne/evoked.py]: I can't try it for myself right now, but ERPs are  almost always in the single-digit uV range - that would mean, either 200e-6 if the data is in V, or somewhere between 2 and 20 if it corresponds to the ylim after scaling the data. I can check later.

**Comment by jasmainak** [comment]
sorry for the noise, I had mistakenly created a branch on upstream. Let's move the discussion here: https://github.com/mne-tools/mne-python/pull/2823/files

**Pull request closed without merging**